src/components/home/PanelResult.vue

```diff
@@ -203,40 +219,44 @@ export default Vue.extend({
   computed: {
     ...mapGetters({
       userGenes: 'getUserGenes',
-      userGenesInPanels: 'getUserGenesInPanels',
       institutionMap: 'getInstitutionMap',
+      panelSearchResult: 'getPanelSearchResult',
+      parsedGenes: 'getParsedGenes'
     }),
   },
+  watch: {
+    panelSearchResult: 'showPanelResult',
+  },
   methods: {
-    getPanelResult() {
-      return this.userGenesInPanels.map((panel: PanelSearchResult) => {
+    showPanelResult() {
+      this.notFoundGenes = this.parsedGenes.notFoundGenes
+      this.panelContent = this.panelSearchResult.map((panel: PanelSearchResult) => {
         const genesInPanel = panel.genesInPanel.map((gene: Gene) =>
-          gene.name.toUpperCase()
+            gene.name.toUpperCase()
         )
         const genesNotInPanel = panel.genesNotInPanel.map((gene: Gene) =>
-          gene.name.toUpperCase()
+            gene.name.toUpperCase()
         )
-
-        let institution = this.institutionMap.get(panel.name)
+        let institution = this.institutionMap.get(panel.name.toUpperCase())
```

Comment:
Remove toUpperCase() for panel name. Only gene name should use it to make easier to compare.

Reply:
i wanted to handle any mismatches happen in the future also so that it wont have comparison issues

Reply:
The issue with dynamic uppercase is that we have to put it everywhere or else we'll have a mismatch. It's best to handle it when we store the institution name if you're worry about being uppercase. It's also possible institutions don't want to be displayed as all caps. Like there's one at UTSW called "Once Upon a Time". That's a bit in your face to be labelled ONCE UPON A TIME. But maybe they don't care. Mostly, we need to be consistent. Since users don't really have to input the institution name, I would rather not have to put uppercase functions all over the place.

Reply:
done